### PR TITLE
LLP-30-MoneyLoundrying

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/styles/money-loundrying.module.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      display: flex;
+      justify-content: center;
+      flex-wrap: wrap;
+      margin-bottom: 80px;
+      padding-bottom: 40px;
+      gap: 30px;
+      cursor: pointer;
+      color: var(--main-text-color);
+      border-bottom: solid 1px var(--table-border-color);
+
```
Comment (pinenetcoder) [inline src/styles/money-loundrying.module.scss:27]: Remove xtra spaces

File: src/styles/money-loundrying.module.scss [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+         &:last-child {
+            margin-right: 0;
+         }
+
+         &.change-language-link {
+            text-transform: uppercase;
+            font-weight: 600;
+         }
+      }
+   }
```
Comment (pinenetcoder) [inline src/styles/money-loundrying.module.scss:48]: Add a space line

File: src/styles/money-loundrying.module.scss (hunk 1 above)
Comment (mdainys) [reply src/styles/money-loundrying.module.scss:27]: Done

File: src/styles/money-loundrying.module.scss (hunk 2 above)
Comment (mdainys) [reply src/styles/money-loundrying.module.scss:48]: Done

**Pull request merged**